FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1899,10 +1920,12 @@
                  "Each element is compared, if the result is true then the resulting element is ~0, else zero"
                 ],
 
+        "ImplicitFlagClobber": true,
         "DestSize": "RegisterSize",
         "NumElements": "RegisterSize / ElementSize"
       },
       "FPR = VFCMPEQ u8:#RegisterSize, u8:#ElementSize, FPR:$Vector1, FPR:$Vector2": {
+        "ImplicitFlagClobber": true,
```

Comment:
This one is an oversight. VFCMPEQ uses fcmeq for SVE which doesn't set flags. Which makes the MMX pfcmpeq worse.

Reply:
nice, fixed!